.github/workflows/helm-ci.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
-          python-version: 3.7
+          python-version: 3.14
```

Comment:
can we use a stable release like 3.13? setup-python [action](https://github.com/grafana/loki/actions/runs/12336549295/job/34450909350?pr=15420) also fails to pick this version

Reply:
Updated